src/dvclive/live.py

```diff
@@ -2,7 +2,9 @@
 import json
 import logging
 import math
+import numpy as np
 import os
+import pandas as pd
```

Comment:
Can we make these optional dependencies and add them here?

https://github.com/iterative/dvclive/blob/84069208144be711ef0feccc38064652d732eac7/pyproject.toml#L46

You can still handle type checking here without always importing those deps: https://stackoverflow.com/questions/61384752/how-to-type-hint-with-an-optional-import